internal/pkg/workspace/workspace.go

```diff
@@ -35,18 +32,25 @@ const (
 	PipelineFileName = "pipeline.yml"
 	// BuildspecFileName is the name of the CodeBuild build specification for the "build" stage of the pipeline.
 	BuildspecFileName = "buildspec.yml"
+	// ManifestFileName is the name of the file that describes the architecture of an application.
+	ManifestFileName = "manifest.yml"
 
 	workspaceSummaryFileName  = ".ecs-workspace"
 	maximumParentDirsToSearch = 5
 	appManifestFileSuffix     = "-app.yml"
```

Comment:
Do you need this anymore?

Reply:
Nope 🕶, I've also moved all the constants to be private except `ProjectDirectoryName`